client/src/helpers/fhir/builder/ObservationBuilder.ts

```diff
@@ -39,6 +39,8 @@ export class ObservationBuilder {
 
     private valueBoolean?: boolean = undefined;
 
+    private valueString?: string = undefined;
```

Comment:
(Optional) I think undefined is just useless here since (to verify) valueString should be implicitly undefined. However, the rest of the code seems to use undefined explicitly.

Reply:
good point. Also, when we defined an 'unsettled' state, its recommended to use null instead of undefined. Since undefined represent a state which JavaScript don't know about it or was not defined. By assigning it to undefined, its no more undefined but defined to undefined 🤔 

JavaScript/Typescript lacking an Optional type, null should be preferred.